app/src/util/networks.ts

```diff
@@ -850,5 +850,12 @@ export const getTxHashBlockExplorerURL = (networkId: number, txHash: string): st
   if (!validNetworkId(networkId)) {
     throw new Error(`Unsupported network id: '${networkId}'`)
   }
-  return `${networks[networkId].blockExplorerURL}${txHash}`
+  return `${networks[networkId].blockExplorerURL}/tx/${txHash}`
+}
+
+export const getContractBlockExplorerURL = (networkId: number, contractAddress: string): string => {
```

Comment:
I know I suggested this name, but maybe better to generalize it to `getAddressBlockExplorerURL` since we could later use it to link to EOAs too